### PR TITLE
Bug issue4 move add receipt image button

### LINKED ISSUE #4: Move add receipt image button
We should move this below `ImgTable` => more space for other util buttons like `show members`

image

And reposition the form like this 

image

Where `ImgTable` is called https://github.com/TueeNguyen/SplitWise3/blob/a3b699ea0efd7a1600b3329c04705d173d35796d/src/web/src/components/expenses/Expense.js#L30

Where it is located [src/components/expenses/addReceiptImg](https://github.com/TueeNguyen/SplitWise3/tree/master/src/web/src/components/expenses/addReceiptImg)

Similar code is [here](https://github.com/TueeNguyen/SplitWise3/blob/a3b699ea0efd7a1600b3329c04705d173d35796d/src/web/src/components/expenses/expenseForms/ReceiptTable.js#L158-L168)

### PR DESCRIPTION
**Pull request by kimjust6**
## Issue This PR Addresses

- Fixes #4 
- UI Improvement

## Description
Moves the add receipt image below and repositions popover to be on the top center wrt to the "Add receipt image" button

image

### CODE REVIEW

File: src/web/src/components/expenses/Expense.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ import ImgTable from './expenseForms/ImgTable';
 import SplitTable from './expenseForms/SplitTable';
 import ReceiptTable from './expenseForms/ReceiptTable';
 import { Form, Formik, FieldArray } from 'formik';
+import AddReceiptButton from '../expenses/addReceiptImg/AddReceiptButton';
+import { colorChannel, width } from '@mui/system';
```
Comment (TueeNguyen) [inline src/web/src/components/expenses/Expense.js:8]: do we use this? Remove

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,4 +1,20 @@
 {
+<<<<<<< HEAD
```
Comment (TueeNguyen) [inline .vscode/settings.json:2]: Please fix merge conflicts

File: src/web/src/components/expenses/Expense.js [hunk 3]
```diff
@@ -30,7 +41,12 @@ const Expense = () => {
   };
   return (
     <div className={classes.container}>
-      <ImgTable />
+      <div className={classes.imgTable}>
+        <ImgTable />
+      </div>
+      <div className={classes.addReceiptButton}>
+        <AddReceiptButton />
+      </div>
```
Comment (TueeNguyen) [inline src/web/src/components/expenses/Expense.js:49]: For code resistancy, can you move this button into `ImgTable`, similar to this
https://github.com/TueeNguyen/SplitWise3/blob/5ab398ffb5e8896afedeee0a8bf31354ac5465c6/src/web/src/components/expenses/expenseForms/ReceiptTable.js#L157-L167

**Review by TueeNguyen — CHANGES_REQUESTED** [review]

**Comment by kimjust6** [comment]
updated as requested.

File: .vscode/settings.json [hunk 4]
```diff
@@ -8,5 +8,8 @@
     "source.fixAll": true
   },
   "files.eol": "\n",
-  "files.insertFinalNewline": true
+  "files.insertFinalNewline": true,
+  "[javascript]": {
+    "editor.defaultFormatter": "vscode.typescript-language-features"
+  }
```
Comment (TueeNguyen) [inline .vscode/settings.json:14]: What problem does this solve?

Comment (TueeNguyen) [inline .vscode/settings.json:14]: What problem(s) do these lines solve?

Comment (kimjust6) [reply .vscode/settings.json:14]: it just chooses the default formatter for the application.

You don't have to merge these if you don't want.

**Comment by TueeNguyen** [comment]
Pettier check didn't pass because I didn't `npm install` first so `pretty-quick` didn't run when commiting.

Note: this is a front-end fix only, will need to move `ImgTable` inside `Formik` component

**Review by TueeNguyen — APPROVED** [review]

**Pull request merged**